### PR TITLE
qa: Added luminous to the mix in schedule_subset.sh

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by joscollin** [comment]
Jenkins retest this please

File: qa/machine_types/schedule_subset.sh [hunk 1]
```diff
@@ -36,6 +36,9 @@ elif [ $2 = "jewel" ] ; then
 elif [ $2 = "kraken" ] ; then
         # run kraken branch with /40 jobs
         teuthology-suite -v -c $2 -m $3 -k distro -s $4 --subset $(echo "(($(date +%U) % 4) * 7) + $1" | bc)/40 -e $5 $6
+elif [ $2 = "luminous" ] ; then
+        # run luminous branch with /40 jobs
+        teuthology-suite -v -c $2 -m $3 -k distro -s $4 --subset $(echo "(($(date +%U) % 4) * 7) + $1" | bc)/40 -e $5 $6
 else
```
Comment (joscollin) [inline qa/machine_types/schedule_subset.sh]: If luminous, kraken and jewel are executing the same statement inside, why can't it be combined in a single `if` ? Or is it intended ?

Comment (tchaikov) [inline qa/machine_types/schedule_subset.sh]: $2

Comment (joscollin) [inline qa/machine_types/schedule_subset.sh]: @tchaikov Ah! got it. Thanks.

**Pull request merged**